pkg/static/login.js

```diff
@@ -164,17 +164,26 @@
     }
 
     function requisites() {
-        function disableLogin(name) {
+        function disableLogin(name, bypass) {
             if (name === "supports")
                 name = "@supports API";
             const errorString = format(_("This web browser is too old to run the Web Console (missing $0)"), name);
 
             if (window.console)
                 console.warn(errorString);
             id("login-error-message").textContent = errorString;
-            hide("#login", "#login-details");
             show("#unsupported-browser", "#error-group");
-            document.body.classList.add("brand-unsupported-browser");
+            document.body.classList.add("unsupported-browser");
+
+            if (bypass) {
+                show("#login", "#login-details");
+                hide("#get-out-link");
+                id("login-button").classList.remove("pf-m-primary");
+                id("login-button").classList.add("pf-m-warning");
+                id("login-button-text").textContent = _("Log in anyway");
+            } else {
```

Comment:
These 7 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17774-20220927-095522-73b8faa5-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)

Reply:
Gonna be interesting trying to figure out how to test that :)

Reply:
Yeah. I don't know how. I guess I could split that chunk out to a function... but it's kind of the most actionable part of a function already. I suppose testing the function it's in with a "bypass" value could work... with pixel tests? But I have no clue how do to most of this.

Reply:
can we override browser defined objects? like `window.replaceAll = undefined` and then this would trigger our check?